modules/command-handler/src/application/types.ts

```diff
@@ -0,0 +1,749 @@
+
+ Contributors
+ --------------
+ This is the official list (alphabetical ordering) of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+ * Modusbox
+ - Shashikant Hirugade <shashikant.hirugade@modusbox.com>
+ - Juan Correa <juancorrea@modusbox.com>
+
+ --------------
+ ******/
+
+"use strict";
+
+import {
+    IBulkTransferRequest,  // TODO: TBD
+    IIndividualTransfers, // TODO: TBD
+    IBulkTransferStatus,
+    IBulkBatch,  // TODO: TBD
+    IBulkTransferOptions,
+    IBulkExtensionList,
+    IExtensionItem,
+    IPartyRequest,
+    IAutoAcceptPartyOption,
+    IAutoAcceptQuoteOption,
+    IPartyIdInfo,
+    IMerchantClassificationCode,
+    IPartyName,
+    IPersonalInfo,
+    IIdType,
+    IFspId,
+    IExtensionList,
+    IPartyComplexName,
+    IDateShort,
+    IBulkTransferEntity  // TODO: TBD mbp-638
+} from "@mojaloop/sdk-scheme-adapater-private-types";
+import {
+    InvalidBulkTransferEntityIdTypeError,
+    InvalidBulkTransferEntityHomeTransactionIDTypeError,
+    InvalidBulkTransferEntityRequestTypeError,
+    InvalidBulkTransferEntityIndividualTransfersTypeError,
+    InvalidBulkTransferEntityStatusTypeError,
+    InvalidBulkTransferEntityBulkBatchTypeError,
+    InvalidBulkTransferEntityPartyLookupTotalCountTypeError,
+    InvalidBulkTransferEntityPartyLookupSuccessCountTypeError,
+    InvalidBulkTransferEntityPartyLookupFailedCountTypeError,
+    InvalidBulkTransferEntityBulkQuotesTotalCountTypeError,
+    InvalidBulkTransferEntityBulkQuotesFailCountTypeError,
+    InvalidBulkTransferEntityBulkTransfersTotalCountTypeError,
+    InvalidBulkTransferEntityBulkTransfersSuccessCountTypeError,
+    InvalidBulkTransferEntityBulkTransfersFailCountTypeError
+} from "./errors";
+
+export class BulkTransferEntity implements IBulkTransferEntity {
+    id: string;
+    bulkHomeTransactionID: string | null;
+    request: BulkTransferRequest;
+    individualtransfers: IndividualTransfers;
+    status: BulkTransferStatus;
+    bulkBatch: BulkBatch;
+    partyLookupTotalCount: number;
+    partyLookupSuccessCount: number;
+    partyLookupFailedCount: number;
+    bulkQuotesTotalCount: number;
+    bulkQuotesSuccessCount: number;
+    bulkQuotesFailedCount: number;
+    bulkTransfersTotalCount: number;
+    bulkTransfersSuccessCount: number;
+    bulkTransfersFailedCount: number;
+
+    constructor(
```

Comment:
You could also just pass the IBulkTransferEntity interface in the constructor.

But lets chat about that on a call.